src/main/java/gregtech/common/metatileentities/multi/electric/generator/LargeCombustionEngineWorkableHandler.java

```diff
@@ -71,4 +76,35 @@ public void deserializeNBT(NBTTagCompound compound) {
         super.deserializeNBT(compound);
         this.currentCycle = compound.getInteger("Cycle");
     }
+
+    @Override
+    protected boolean isObstructed() {
+        return checkIntakesObstructed();
```

Comment:
The intake check method could be put in here directly.

Reply:
Going to leave it separate so that the logic can be easily added to in case we want to add another obstruction mechanic at any time.